packages/kbn-config-schema/tsconfig.json

```diff
@@ -1,14 +1,13 @@
 {
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
-    "incremental": false,
-    "outDir": "./target/out",
-    "declarationDir": "./target/types",
-    "stripInternal": true,
     "declaration": true,
     "declarationMap": true,
+    "outDir": "target",
```

Comment:
are we going to use `bazel` caching mechanism instead of tsc incremental builds for all the projects?

Reply:
I plan to use both https://github.com/elastic/kibana/pull/96273/files#diff-e62f86d0357aeca181de69ece188b14fc09bf49209b7ee354499eba411a240c2R56

I forgot to be explicit on the tsconfig because it was already the default but I'll add it 👍